source/includes/_notifications.md

```diff
@@ -405,6 +414,130 @@ event_uri | string  | The event_uri for the desired event
 
 
 
+## Send Notification
+
+```shell
+# Get your token for further authorization
+curl -X GET "https://core.eventtia.com/v1/events/<event_uri>/notifications/<id>/deliver" \
+  -H 'Content-Type: application/json'
+```
+
+```javascript
+fetch('https://core.eventtia.com/v1/events/<event_uri>/notifications/<id>/deliver', {
+  method: 'GET',
+  headers: {
+    'Authorization': 'Bearer <your token>',
+  }
+})
+```
+
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Make sure you replace &lt;event uri&gt; with the event uri for the event.
+
+> Make sure you replace &lt;id&gt; with the id for the notification to obtain. 
+
+> Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "status": 200
+}
+```
+
+>Example of Not Found (404) response: 
+
+```http
+HTTP/1.1 404 Not Found
+{
+  {
+    "message": "Couldn't find Notification"
+  }
+}
+```
+
+This endpoint sends mass notification and return it
+
+***HTTP Request***
+
+`GET /v1/events/:event_uri/notifications/:id/deliver`
+
+***Path Parameters***
+
+Parameter |  Type   | Description
+--------- | ------- | -----------
+event_uri | string  | The event_uri for the desired event
+   id     | integer | The id for the desired notification
+
+
+## Stats Notification
+
+```shell
+# Get your token for further authorization
+curl -X GET "https://core.eventtia.com/v1/events/<event_uri>/notifications/<id>/stats" \
+  -H 'Content-Type: application/json'
+```
+
+```javascript
+fetch('https://core.eventtia.com/v1/events/<event_uri>/notifications/<id>/stats', {
+  method: 'GET',
+  headers: {
+    'Authorization': 'Bearer <your token>',
+  }
+})
+```
+
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Make sure you replace &lt;event uri&gt; with the event uri for the event.
+
+> Make sure you replace &lt;id&gt; with the id for the notification to obtain. 
+
+> Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "meta": {
+    "status": "sent",
+    "stats": {
+        "sent": 1,
+        "error_logs": 0,
+        "total_recipients": null
+    },
+    "error_logs": [
+        "error proccessing recipient: 1, error SMTP To address may not be blank: []",
+        "error proccessing recipient: 2, error SMTP To address may not be blank: []"
+    ]
+  }
+}
+```
+
+>Example of Not Found (404) response: 
+
+```http
+HTTP/1.1 404 Not Found
+{
+  {
+    "message": "Couldn't find Notification"
+  }
+}
+```
+This endpoint get the mass notifications stats and return it
```

Comment:
This endpoint allows you to obtain the progress and stats of a massive notification after the user sent it.

Reply:
Done